cmd/cluster/azure/create.go

```diff
@@ -48,6 +49,11 @@ func bindCoreOptions(opts *RawCreateOptions, flags *pflag.FlagSet) {
 	flags.StringVar(&opts.DiskStorageAccountType, "disk-storage-account-type", opts.DiskStorageAccountType, "The disk storage account type for the OS disks for the VMs.")
 	flags.StringToStringVarP(&opts.ResourceGroupTags, "resource-group-tags", "t", opts.ResourceGroupTags, "Additional tags to apply to the resource group created (e.g. 'key1=value1,key2=value2')")
 	flags.StringVar(&opts.SubnetID, "subnet-id", opts.SubnetID, "The subnet ID where the VMs will be placed.")
+	flags.StringVar(&opts.MarketplacePublisher, "marketplace-publisher", opts.MarketplacePublisher, "The Azure Marketplace image publisher.")
```

Comment:
based on the code below if MarketplacePublisher != nil all the other new fields are required? where is that communicated?

Reply:
It's been added here - https://github.com/openshift/hypershift/blob/82c92662715c65de4e9b6c855c71e23a68ac88a2/cmd/cluster/azure/create.go#L114